packages/css/src/components/accordion/accordion.scss

```diff
@@ -34,20 +40,21 @@
   }
 
   &:hover {
-    box-shadow: var(--ams-accordion-button-hover-box-shadow);
+    color: var(--ams-accordion-button-hover-color);
   }
+}
 
-  svg {
-    transform: rotate(0deg);
-    transition: transform 0.3s ease;
+.ams-accordion__icon svg {
+  transform: rotate(0deg);
+  transition: transform 0.3s ease;
 
-    @media (prefers-reduced-motion) {
-      transition: none;
-    }
+  [aria-expanded="true"] & {
+    transform: rotate(180deg);
```

Comment:
Now that the chevron is at the start of the line, rotating clockwise feels more natural.